knowage-core/src/main/java/it/eng/spagobi/sdk/utilities/SDKObjectsConverter.java

```diff
@@ -228,8 +229,11 @@ public ObjTemplate fromSDKTemplateToObjTemplate(SDKTemplate sdkTemplate) {
 			}
 			if (dh != null && dh.getName() != null) {
 				LOGGER.debug("Deleting attachment file ...");
-				File attachment = new File(dh.getName());
-				if (attachment.exists() && attachment.isFile()) {
+				String fileName = dh.getName();
+				PathTraversalChecker.isValidFileName(fileName);
```

Comment:
fileName seems to be a complete path instead of a single file name. Please check

Reply:
We need to talk about this in today's call.

Reply:
revert the changes, most likely this is a false positive